tests/test_core.py

```diff
@@ -8129,13 +8129,11 @@ def test_pthread_exceptions(self):
   def test_emscripten_atomics_stub(self):
     self.do_run_in_out_file_test('tests', 'core', 'pthread', 'emscripten_atomics.c')
 
-  @no_asan('incompatibility with atomics')
   @node_pthreads
   def test_emscripten_atomics(self):
     self.set_setting('USE_PTHREADS', '1')
     self.do_run_in_out_file_test('tests', 'core', 'pthread', 'emscripten_atomics.c')
 
-  @no_asan('incompatibility with atomics')
   @node_pthreads
```

Comment:
asan only runs on chromium CI after commit, so it's not tested here, but I assume you saw it works?